get_wlat.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
```

Comment:
Not entirely sure what you wanna get with this script, but I have the impression that some of this code can be simplified quite a lot by using this existing API:
https://github.com/ARM-software/lisa/blob/master/libs/utils/analysis/latency_analysis.py

A usage example is reported here:
https://github.com/ARM-software/lisa/blob/master/ipynb/examples/trace_analysis/TraceAnalysis_TasksLatencies.ipynb

Notice that you can get the DF of latency events for each task you want and then just describe() it to get the stats you are after.